.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## Related Issue
+*Please link to the issue this pull request addresses. If there isn't one yet, please file one
+prior to creating the pull request so that it can be discussed.*
+
+## Testing
+*Please describe what you have done to test your changes.*
+
+## Checklist
+- [ ] I have read the [Arcade Contributor Guidance doc](https://github.com/dotnet/arcade/blob/master/Documentation/Policy/ArcadeContributorGuidance.md)
+- [ ] Your code builds successfully
```

Comment:
The PR build and checks will catch this.

Reply:
Actually yeah this should be "all checks pass" i.e. don't merge on a failed build.